### DISCUSSION TITLE
Cart Quantity Update.

### DISCUSSION THREAD

**Discussion by Rub-s** [vanilophp/framework#158] [Q&A]
I checked the code and Cart tests.
I don't find the process in the current state to update cart item quantity. Nighther in code , doc or test folder.
There are these methods : GetQuantity, AddItem.

Also i checked in Tests folder , it seems there are no test done on changing cart item Qty.

Did i missed something ?

**Comment by fulopattila122** [comment]
You can just simply update the quantity field and save the model. All models in Vanilo are Eloquent models.

Eg.:

```php
$cartItem->quantity = 3;
$cartItem->save();
```

Example: https://github.com/vanilophp/demo/blob/master/app/Http/Controllers/CartController.php#L52
